### PR TITLE
THU/scenario_manager_tree_list_prod_7333

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: src/ScenarioManagerTreeList/ScenarioManagerTreeList.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      id: scenario.id,
+      name: scenario.name,
+      title: <ScenarioNode datasets={datasets} scenario={scenario}/>
+    };
+  });
+  return ScenarioUtils.getScenarioTree(rstscenarios,
+    (scenA, scenB) => (scenA.name.localeCompare(scenB.name)));
+}
+
+function getScenariosCount (treeData) {
```
Comment (jreynard-code) [inline src/ScenarioManagerTreeList/ScenarioManagerTreeList.js:109]: Can be moved into the core package ;)

File: src/ScenarioManagerTreeList/ScenarioManagerTreeList.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const {
+    datasets,
+    scenarios,
+    moveScenario
+  } = props;
+
+  const [searchText, setSearchText] = useState('');
+  const [treeData, setTreeData] = useState(formatScenariosToRSTList(datasets, scenarios));
```
Comment (jreynard-code) [inline src/ScenarioManagerTreeList/ScenarioManagerTreeList.js:24]: Could be interesting to use "useMemo" hooks here:

`const rstList = useMemo(() => formatScenariosToRSTList(datasets, scenarios), [datasets,scenarios]);
const [treeData, setTreeData] = useState(rstList); `

and in `filterScenarios` function

`  if (!searchStr || searchStr.length === 0) {
      setTreeData(rstList);
      return;
    }`

File: src/ScenarioManagerTreeList/components/ScenarioNode/ScenarioNode.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  if (scenarioState === 'Successful') {
+    return classes.statusSuccessful;
+  }
+  if (scenarioState === 'Failed') {
+    return classes.statusFailed;
+  }
+  return classes.statusCreated;
+}
+
+function getDatasetNames (datasets, scenarioDatasets) {
```
Comment (jreynard-code) [inline src/ScenarioManagerTreeList/components/ScenarioNode/ScenarioNode.js:78]: Can be moved into the core package ;)

**Review by jreynard-code — CHANGES_REQUESTED** [review]
Minor changes, LGTM ;)

**Comment by csm-thu** [comment]
@jreynard-code suggested changes have been done, thanks for the feedback ! :+1:

**Pull request merged**